houston/deployment.go

```diff
@@ -233,6 +239,38 @@ var (
 		},
 	}
 
+	PaginatedDeploymentsGetRequest = queryList{
```

Comment:
This is mixing spaces and tabs, can we just use tabs? Looks like the other requests use tabs.

Reply:
Done!